File: src/Entity/Livreur.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\LivreurRepository;
use App\Entity\Utilisateur;
use ORM\Table;

 
#[ORM\Entity(repositoryClass: LivreurRepository::class)]

class Livreur
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id_livreur = null ;
    

    #[ORM\Column(length:50)]
    private ?string $cinLivreur = null ;
   

    #[ORM\Column(length:50)]
    private ?string $nom = null ;
    


    #[ORM\Column(length:50)]
    private ?string $prenom = null ;
   
    

    #[ORM\Column(length:50)]
    private ?string $vehicule = null ;


    public function getIdLivreur(): ?int
    {
        return $this->id_livreur;
    }

    public function getCinLivreur(): ?string
    {
        return $this->cinLivreur;
    }

    public function setCinLivreur(string $cinLivreur): self
    {
        $this->cinLivreur = $cinLivreur;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getVehicule(): ?string
    {
        return $this->vehicule;
    }

    public function setVehicule(string $vehicule): self
    {
        $this->vehicule = $vehicule;

        return $this;
    }


}

File: src/com/mycompany/myapp/entities/Livreur.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author amens
 */
public class Livreur {

    int idlivreur;
    String cinLivreur, nom, prenom, vehicule;

    public Livreur() {
    }

    public Livreur(String cinLivreur, String nom, String prenom, String vehicule) {
        this.cinLivreur = cinLivreur;
        this.nom = nom;
        this.prenom = prenom;
        this.vehicule = vehicule;
    }

    public Livreur(int idlivreur, String cinLivreur, String nom, String prenom, String vehicule) {
        this.idlivreur = idlivreur;
        this.cinLivreur = cinLivreur;
        this.nom = nom;
        this.prenom = prenom;
        this.vehicule = vehicule;
    }

    public int getIdlivreur() {
        return idlivreur;
    }

    public void setIdlivreur(int id_livreur) {
        this.idlivreur = id_livreur;
    }

    public String getCinLivreur() {
        return cinLivreur;
    }

    public void setCinLivreur(String cinLivreur) {
        this.cinLivreur = cinLivreur;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getVehicule() {
        return vehicule;
    }

    public void setVehicule(String vehicule) {
        this.vehicule = vehicule;
    }

    @Override
    public String toString() {
        return "Livreur{" + "id_livreur=" + idlivreur + ", cinLivreur=" + cinLivreur + ", nom=" + nom + ", prenom=" + prenom + ", vehicule=" + vehicule + '}';
    }

    
}
